test/integration_gcp_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build gcp
+//go:build gcp || awsgcp
```

Comment:
NIT: Do we need to also run the awsgcp tests here? Seems like a waste of resources, given that we're going to run `aws` tests on their own anyways.

Reply:
`awsgcp` this means that we need helper funcs from this and other files.